.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+name: CICD
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  run-lint:
+    runs-on: ubuntu-latest
+    name: Run linter
+    steps:
```

Comment:
Why did you add `main.yml`?
The default branch is not even "main", I think this file is not necessary for now, did line up something with a mantainer: